plugins/newsdownloader.koplugin/feed_config.lua

```diff
@@ -1,11 +1,27 @@
 return {
 	-- list your feeds here:
 
-    { "http://feeds.reuters.com/Reuters/worldNews?format=xml", limit = 2},
-    -- set 'limit' to change number of 'news' to be downloaded from source
+    { "http://feeds.reuters.com/Reuters/worldNews?format=xml", limit = 2, download_full_article=false},
+
+    { "https://www.pcworld.com/index.rss", limit = 7 , download_full_article=true},
```

Comment:
You could add https://github.com/koreader/koreader/releases.atom as an example of a feed that doesn't require downloading the full article.

Reply:
too boring:P